mono/metadata/icall-def.h

```diff
@@ -439,6 +439,31 @@ ICALL(MATH_16, "Sqrt", ves_icall_System_Math_Sqrt)
 ICALL(MATH_17, "Tan", ves_icall_System_Math_Tan)
 ICALL(MATH_18, "Tanh", ves_icall_System_Math_Tanh)
 
+ICALL_TYPE(MATHF, "System.MathF", MATHF_1)
+ICALL(MATHF_1, "Acos", ves_icall_System_MathF_Acos)
+ICALL(MATHF_2, "Acosh", ves_icall_System_MathF_Acosh)
+ICALL(MATHF_3, "Asin", ves_icall_System_MathF_Asin)
+ICALL(MATHF_4, "Asinh", ves_icall_System_MathF_Asinh)
+ICALL(MATHF_5, "Atan", ves_icall_System_MathF_Atan)
+ICALL(MATHF_6, "Atan2", ves_icall_System_MathF_Atan2)
+ICALL(MATHF_7, "Atanh", ves_icall_System_MathF_Atanh)
+ICALL(MATHF_8, "Cbrt", ves_icall_System_MathF_Cbrt)
+ICALL(MATHF_9, "Ceiling", ves_icall_System_MathF_Ceiling)
+ICALL(MATHF_10, "Cos", ves_icall_System_MathF_Cos)
+ICALL(MATHF_11, "Cosh", ves_icall_System_MathF_Cosh)
+ICALL(MATHF_12, "Exp", ves_icall_System_MathF_Exp)
+ICALL(MATHF_22, "FMod", ves_icall_System_MathF_FMod)
+ICALL(MATHF_13, "Floor", ves_icall_System_MathF_Floor)
+ICALL(MATHF_14, "Log", ves_icall_System_MathF_Log)
+ICALL(MATHF_15, "Log10", ves_icall_System_MathF_Log10)
+ICALL(MATHF_23, "ModF(float,float&)", ves_icall_System_MathF_ModF)
```

Comment:
This signature doesn't match the managed signature.

Reply:
Good catch, thanks!

Reply:
I changed my mind, I kept the .NET signature.

Reply:
Short version: done